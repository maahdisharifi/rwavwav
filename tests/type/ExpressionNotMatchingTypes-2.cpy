int main():
    int x = 1
    float y = 1.1
    int m = x + y
    float n = x + y
    return 0
end