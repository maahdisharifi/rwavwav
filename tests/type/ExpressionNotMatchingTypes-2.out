Line:4 -> type mismatch in expression